backend/app.py

```diff
@@ -28,6 +29,18 @@ def handle_users():
     else:
         return jsonify("METHOD NOT ALLOWED"), 405
 
+
+#####################################################################
+#                               STOCK                               #
+#####################################################################
+
+@app.route('/stocker/stock/savedStocks', methods=["GET"])
```

Comment:
I believe the userid parameter must be attached at the end of the route.

Reply:
done